modules/social_features/social_profile/modules/social_profile_organization_tag/social_profile_organization_tag.install

```diff
@@ -15,6 +15,7 @@ function social_profile_organization_tag_install() {
     [
       'delete terms in profile_organization_tag',
       'edit terms in profile_tag',
+      'create terms in profile_tag',
```

Comment:
@Macsym2018 Since we set `profile_tag` in the `social_profile.install` here looks like we have old mistake and we should fix everywhere to `profile_organization_tag`